learn/README.md

```diff
@@ -0,0 +1,106 @@
+# Pinecone.io Example Jupyter Notebooks
+
+![Pinecone Logo](./images/pinecone_logo_black.png)
+![Long term memory for Artificial Intelligence](./images/long-term-memory-for-ai.jpeg)
+
+# What is this repo?
+
+This repository is a collection of Jupyter Notebooks that you can run, download, study and modify in order to get hands-on with 
+Pinecone.io vector databases and common Machine Learning patterns, tools and algorithms.
+
+These examples are ideal for learning and expanding your knowledge of what's possible, but if you're looking for Production-ready examples, please see the [docs](../docs) folder.
+
+Each individual example notebook lives in its own directory. A notebook file ends in `.ipynb`, and each individual Notebook demonstrates a different Machine Learning technique or usage pattern. 
+
+# Getting started 
+
+Running the examples in this repo requires two things, both of which are free: 
+1. a Pinecone.io account, represented by an API key
+1. a place to run the example Notebooks. In this case, we'll be using Google Colaboratory
+
+## Step 1. Create a Pinecone.io account. 
+
+These example notebooks require Pinecone.io API key to run, because they create and query Pinecone vector database indexes.
+
+Pinecone.io offers a generous free tier that's ideal for learning and experimenting, and even building applications that leverage vector databases.
+
+Head over to [the Pinecone.io dashboard](https://app.pinecone.io/) and create a new account, which you can do in under a minute. 
+
+## Step 2. Get your Pinecone.io API key
+
+On the left-hand side of the Pinecone dashboard, click the API Keys header. You'll have a default API key already available. **_Your API Key is a secret - don't share it or commit it to version control_!**
+
+![Get your Pinecone.io API key](./images/pinecone-api-keys.png)
+
+## Step 3. Run a notebook via Google Colaboratory
+
+While logged into a Google account, visit [Google Colaboratory](https://colab.research.google.com/). This is a Google service which allows you to run Jupyter Notebooks on Google infrastructure to greatly simplify experimenting, exploring and sharing your results 
+without needing to worry about creating or managing cloud compute environments.
+
+## Step 4. Perform the one-time setup for Google Colaboratory 
+
+You will only need to complete these next steps one time, and can then skip them in the future when running additional example Notebooks.
+
+Start by clicking File > Open Notebook in the top left menu of Google Colab and be sure to click the GitHub tab. You can then either: 
+
+1. Paste the full URL (including the `.ipynb` extension) into the URL bar in the modal, or you can enter `pinecone-io` (which is the Pinecone.io GitHub organization name) and then enter `pinecone-io/examples` in the repository field: 
```

Comment:
people can also just click the "Colab" button at the top of most notebooks (but not all, so these instructions are valid for where we might be missing that — but we should aim to have colab shields in all notebooks to minimize friction)

Reply:
Good call and will add - one thing I noticed yesterday toward EOD is that some of the Colab buttons 404'd / didn't load properly, but I'll need to dig in a bit more to figure out what's going on.

Reply:
Added in [b82957b](https://github.com/pinecone-io/examples/pull/240/commits/b82957b861fe9916f717d96d066fd0ab9f4cb5f6)